manifests/init.pp

```diff
@@ -15,6 +15,7 @@
   Array[String] $activemq_hosts                = [ '127.0.0.1' ],
   Integer       $activemq_port                 = 8161,
   Boolean       $symlink_puppet_metrics_collector = true,
+  Optional[Puppet_metrics_collector::Metrics_server] $metrics_server_info = undef,
```

Comment:
The format for this parameter needs to be documented in the readme as well as an example.